cmd/svcat/touch/touch_instance_cmd.go

```diff
@@ -0,0 +1,45 @@
+package touch
+
+import (
+	"fmt"
+
+	"github.com/kubernetes-incubator/service-catalog/cmd/svcat/command"
+	"github.com/spf13/cobra"
+)
+
+type touchInstanceCmd struct {
+	*command.Context
+	namespace string
+	name      string
+}
+
+func newTouchInstanceCmd(cxt *command.Context) *cobra.Command {
+	touchInstanceCmd := &touchInstanceCmd{Context: cxt}
+	cmd := &cobra.Command{
+		Use:   "touch NAME --namespace NAMESPACE",
+		Short: "touch an instance to make service-catalog try to provision again",
```

Comment:
`s/provision/process the spec`?
i.e. the result of `updateRequests++` could be update, delete or possibly even no-op, not just provisioning.